src/server.ts

```diff
@@ -1,4 +1,5 @@
 import { Pinecone } from '@pinecone-database/pinecone';
+import fetch from 'node-fetch';
```

Comment:
You don't need that!  `fetch` is now baked directly into node.js, and we don't need that package anymore.